stan/math/prim/mat/prob/bernoulli_logit_glm_lpmf.hpp

```diff
@@ -107,7 +110,7 @@ typename return_type<T_x, T_alpha, T_beta>::type bernoulli_logit_glm_lpmf(
   T_partials_return theta_derivative_sum = 0;
   T_partials_return exp_m_ythetan;
   for (size_t n = 0; n < N; ++n) {
-    ytheta[n] += signs[n] * value_of(alpha_vec[n]);
+    check_finite(function, "Matrix of independent variables", ytheta[n]);
```

Comment:
Is this the error message we want to pass back if it's not finite? Have you been able to trigger this condition?